package.json

```
@@ -59,7 +59,7 @@
     "release-it-lerna-changelog": "^1.0.2"
   },
   "engines": {
-    "node": "6.* || 8.* || 9.* || >= 10.*"
+    "node": "8.* || 9.* || >= 10.*"
```

Let’s go with:

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```

9 and 11 are unsupported by the node folks also